README.md

```diff
@@ -15,23 +15,52 @@ chmod +x /usr/local/bin/fwd
 * For Release Candidate and testing the latest features please checkout https://github.com/fireworkweb/fwd/issues/70#issuecomment-554606414.
 * Don't have and don't want to have PHP installed locally? Checkout and follow https://github.com/fireworkweb/fwd/issues/21 for 100% Docker usage, thus not requiring PHP.
 
-## Usage
+## Getting started
 
-```bash
-# to install docker-compose.yml file into the project (just the first time)
+#### Use in a new project
+
+`fwd` is great for running Docker Compose based projects. If you wanna use it in your project simply go to the root folder and use:
+
+```base
 fwd install
+```
+
+This will create the `.fwd` (environment variables) and `docker-compose.ym` (services definitions) files in your project (you probably want to have them versioned).
```

Comment:
```suggestion
This will create the `.fwd` (environment variables) and `docker-compose.yml` (services definitions) files in your project (you probably want to have them versioned).
```